### ISSUE TITLE
How to change the background color of the text typed in the snap?

### ISSUE THREAD

**Issue by ixpress** [Snipaste/feedback#1158]
This maybe a feature request:
I need to change the background color of the text(or fill the back ground with color), so that it's clear, not be mixed with the background together.
Check the image for detail:
Hardly to see:
<img width="722" alt="Snipaste_2019-06-25_11-46-52" src="https://user-images.githubusercontent.com/1673857/60068594-017b3c00-9742-11e9-9bf5-17ebcc943f4f.png">
Better:
<img width="682" alt="Snipaste_2019-06-25_11-47-59" src="https://user-images.githubusercontent.com/1673857/60068607-09d37700-9742-11e9-8624-64696a4adf81.png">

**Comment by liulex** [comment]
Yes, background color of text is not supported yet, but will be added in near future.
For now you can enable text outline to make the text more readable.

**Comment by kourosious** [comment]
Is there no hope no more for this option? :)
